### PR TITLE
1deg jra55do ryf.cherrypicks

### PR DESCRIPTION
**Pull request by anton-seaice**
Companion PR to https://github.com/COSIMA/MOM6-CICE6/pull/91

The input folder was been renamed to INPUT, and now WW3 can't find the restart file :

```
Model definition file read.

 *** WAVEWATCH III WARNING IN W3IORS : 
     NO READABLE RESTART FILE, INITIALIZE WITH 'CALM' INSTEAD
     IOSTAT =   29

 Restart file read; cold start (calm).
```

Suggestions @ezhilsabareesh8  ?

### CODE REVIEW

**Comment by dougiesquire** [comment]
@anton-seaice can you please remove the first commit ([db244fa](https://github.com/COSIMA/MOM6-CICE6-WW3/pull/44/commits/db244fa01d405100610313a641b36f7ac4b9ebe1))? We haven't tested this yet so I don't think we want to be propagating it across our configs - see [my comment](https://github.com/COSIMA/MOM6-CICE6/pull/94#issuecomment-2272894513) in the original PR.

In fact, I think we should revert this commit from `MOM6-CICE6/1deg_jra55do_ryf` until this appropriate testing is done - see [this comment](https://github.com/COSIMA/MOM6-CICE6/pull/47#issuecomment-2301672105).

**Comment by ezhilsabareesh8** [comment]
@anton-seaice The input directory name in `wav_in` has to be changed. 

[initfile = "./input/restart.ww3"](https://github.com/COSIMA/MOM6-CICE6-WW3/blob/bf90953b8669d82aeb6ad0646ad0e15ee8d22229/wav_in#L6-L7)

**Comment by anton-seaice** [comment]
This is ready to go.

File: nuopc.runconfig [hunk 1]
```diff
@@ -287,7 +287,7 @@ CLOCK_attributes::
      history_ymd = -999
      ice_cpl_dt = 99999 #not used
      lnd_cpl_dt = 99999 #not used
-     ocn_cpl_dt = 3600
+     ocn_cpl_dt = 1800 #ignored (coupling timestep set by nuopc.runseq) unless stop_option is nsteps
```
Comment (minghangli-uni) [inline nuopc.runconfig:290]:
```suggestion
     ocn_cpl_dt = 3600 #ignored (coupling timestep set by nuopc.runseq) unless stop_option is nsteps
```

Comment (anton-seaice) [reply nuopc.runconfig]: Thanks!

**Review by dougiesquire — APPROVED** [review]
Thanks @anton-seaice. LGTM and seems to run successfully as is.

**Pull request merged**